components/Menu.tsx

```diff
@@ -9,16 +12,35 @@ export default function Menu() {
         await router.push(route);
     };
 
+    const isHomeRoute = () => {
+        const currentPath = window.location.pathname;
+        const homePath = `/${Routes.HOME.toLowerCase()}`;
+        return currentPath === homePath;
+    };
+    const [showMenuContent, setShowMenuContent] = useState(isHomeRoute());
+
+    useEffect(() => {
+        if (isHomeRoute()) {
+            setShowMenuContent(true);
+        } else {
+            setShowMenuContent(false);
+        }
+    }, [showMenuContent]);
+
```

Comment:
Remove all this "isHomeRoute" functionality and just add the Dashboard component to home/page.tsx html. It makes code cleaner and Dashboard is only needed on home page.